webviews/scripts/main.js

```
@@ -0,0 +1,249 @@
+    }
+    return hljs.highlightAuto(code).value;
+  },
+  breaks: true,
+  gfm: true,
+});
+
+window.addEventListener("message", (event) => {
+  const message = event.data;
+  switch (message.command) {
+    case 'chatResponse':
+      const thinkingBubble = document.getElementById(message.bubbleId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)